ui/page/privacy/manual_mixer_setup_page..go

```diff
@@ -0,0 +1,224 @@
+						layout.Flexed(1, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									return pg.mixerAccountSections(gtx, "Mixed account", pg.mixedAccountSelector.Layout)
+								}),
+								layout.Rigid(func(gtx C) D {
+									return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
+										return pg.mixerAccountSections(gtx, "Unmixed account", pg.unmixedAccountSelector.Layout)
+									})
+								}),
+								layout.Rigid(func(gtx C) D {
+									return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16, Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+										return layout.Flex{
+											Axis: layout.Horizontal,
+										}.Layout(gtx,
+											layout.Rigid(func(gtx C) D {
+												return pg.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
+											}),
+											layout.Rigid(func(gtx C) D {
+												txt := `<span style="text-color: grayText2">
+											<b>Make sure to select the corresponding accounts to the previous setup. </b> Failing to do so could damage wallet privacy.
+										</span>`
+												return layout.Inset{
+													Left: values.MarginPadding8,
+												}.Layout(gtx, renderers.RenderHTML(txt, pg.Theme).Layout)
+											}),
+										)
+									})
+								}),
+							)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return layout.UniformInset(values.MarginPadding15).Layout(gtx, pg.toPrivacySetup.Layout)
+						}),
+					)
+				})
+			},
+		}
+		return page.Layout(gtx)
+	}
+
+	return components.UniformPadding(gtx, body)
+}
+
+func (pg *ManualMixerSetupPage) mixerAccountSections(gtx layout.Context, title string, body layout.Widget) layout.Dimensions {
+	return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+		return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Inset{
+						Bottom: values.MarginPadding8,
+					}.Layout(gtx, pg.Theme.Body1(title).Layout)
+				}),
+				layout.Rigid(body),
+			)
+		})
+	})
+}
+
+func (pg *ManualMixerSetupPage) showModalSetupMixerAcct() {
+	if pg.mixedAccountSelector.SelectedAccount().Number == pg.unmixedAccountSelector.SelectedAccount().Number {
+		pg.Toast.NotifyError("Cannot use same account for mixed & unmixed")
+		return
+	}
+
+	if pg.mixedAccountSelector.SelectedAccount().Name == "mixed" || pg.mixedAccountSelector.SelectedAccount().Name == "unmixed" {
+		pg.Toast.NotifyError("The selected mixed account already has privacy enabled")
+		return
+	}
+
+	if pg.unmixedAccountSelector.SelectedAccount().Name == "mixed" || pg.unmixedAccountSelector.SelectedAccount().Name == "unmixed" {
+		pg.Toast.NotifyError("The selected unmixed account already has privacy enabled")
+		return
+	}
```

Comment:
For auto mixer, if there is a wallet account with the name mixed, this shows up.
<img width="517" alt="image" src="https://user-images.githubusercontent.com/27733432/159501866-568db8cf-ce6c-44ba-b111-d87dec7582e8.png">

However, in the case of the manual set up, i get this error and its misleading because i currently don't have mixer setup yet.
<img width="791" alt="image" src="https://user-images.githubusercontent.com/27733432/159502124-24fe211c-11d0-4eda-acd5-6a47b5f99ccd.png">

There is a mixer method "AccountMixerConfigIsSet` that should be able to tell if the wallet has mixer enabled. if it is not and its just a wallet account name, i believe the error modal from the auto mixer will be a better fit for this rather than this toast error.

Reply:
that check isn't meant to be done in the first place, i added it in the initial stages of testing when implementing the functionality, it would be removed in my next commit

manual cspp doesn't mind if you have a wallet named mixed or unmixed